### PR TITLE
Fix Main tab - clicking the message multiple times quickly results in multiple Task Modules opening.

### PR DESCRIPTION
**Pull request by amyzhao1321**
Fix Main tab - clicking the message multiple times quickly results in multiple Task Modules opening.

### CODE REVIEW

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/DraftMessages/draftMessages.tsx [hunk 1]
```diff
@@ -46,11 +46,12 @@ export interface IMessageState {
 
 class DraftMessages extends React.Component<IMessageProps, IMessageState> {
   private interval: any;
+  private openTaskModule: boolean;
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/DraftMessages/draftMessages.tsx]: Change this and variable in messages.tsx to "isOpenTaskModuleAllowed"

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/DraftMessages/draftMessages.tsx [hunk 2]
```diff
@@ -150,21 +151,25 @@ class DraftMessages extends React.Component<IMessageProps, IMessageState> {
...
-      });
-    };
+    if (this.openTaskModule) {
+      this.openTaskModule = false;
+      let taskInfo: ITaskInfo = {
+        url: url,
+        title: title,
+        height: 530,
+        width: 1000,
+        fallbackUrl: url
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/DraftMessages/draftMessages.tsx]: Trailing comma

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx [hunk 3]
```diff
@@ -214,18 +214,22 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
...
-    let submitHandler = (err: any, result: any) => {
-    };
+    if (this.openTaskModule) {
+      this.openTaskModule = false;
+      let taskInfo: ITaskInfo = {
+        url: url,
+        title: title,
+        height: 530,
+        width: 1000,
+        fallbackUrl: url
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx]: Trailing comma

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/DraftMessages/draftMessages.tsx (hunk 1 above)
Comment (amyzhao1321) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/DraftMessages/draftMessages.tsx]: done

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/DraftMessages/draftMessages.tsx (hunk 2 above)
Comment (amyzhao1321) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/DraftMessages/draftMessages.tsx]: done

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx (hunk 3 above)
Comment (amyzhao1321) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx]: done

**Pull request merged**